src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
+
+  let sum = 0;
+
+  men.forEach((m) => sum += m.died - m.born);
```

Comment:
The reduce() function is the best option when you need to calculate a value, i.e. reduce a set into a single value after a specific computation
You can check the [reference docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce) if you want

In this case, you can replace it for
```suggestion
const sum = men.reduce((accumulator, man) => accumulator + (man.died - man.born), 0);
```
Above I replaced the variable named 'm' to 'man' to make it more descriptive